src/pages/_error.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Rollbar from 'rollbar';
+
+function Error({ statusCode }) {
+  return (
+    <p>
+      {statusCode
+        ? `An error ${statusCode} occurred on server!`
+        : 'An error occurred on client'}
+    </p>
+  );
+}
+
+Error.getInitialProps = ({ req, res, err }) => {
+  let statusCode;
+  if (res) {
+    statusCode = res.statusCode;
+  } else if (err) {
+    statusCode = err.statusCode;
+  } else {
+    statusCode = 404;
+  }
+
+  // Only require Rollbar and report error if we're on the server & it's not a local 404.
+  if (!process.browser && statusCode !== 404) {
+    const rollbar = new Rollbar({
```

Comment:
Just to check… will this handle server side errors for code that executes in the `getServerSideProps` functions?

Reply:
Yep - thats exactly what it's for. It might duplicate some of the error reporting you've already handled, but for example I tested it by changing the call on the privacy page to be /privac rather than /privacy and it reports that as an error.

Reply:
Sounds good!